Library/Homebrew/official_taps.rb

```diff
@@ -8,6 +8,14 @@
   tex
 ].freeze
 
+OFFICIAL_CASK_TAPS = %w[
```

Comment:
I'm hesitant to add any more of these as it slows down every search for each additional one we add in here (and I'm currently actively trying to remove Homebrew's ones)